### ISSUE TITLE
Hangs up on login

### ISSUE THREAD

**Issue by udion** [brave/browser-laptop#15196]
I am using brave on linux (Ubuntu 16.10)

I have noticed that the sites which require the login using facebook/google/other_accounts usually opens a pop up window for login to facebook/google/other_accounts. The problem is that the pop-window hangs after putting in the credentials.
Any quick hack to resolve this?

Here are some screenshots

[Soundcloud site](https://soundcloud.com/)
![image](https://user-images.githubusercontent.com/18963158/45481009-0dc07a80-b768-11e8-8a38-921a2c94399b.png)

After clicking the sign-in
![image](https://user-images.githubusercontent.com/18963158/45481036-2597fe80-b768-11e8-80ee-2725774c53de.png)

After selecting google and selecting the account, this window hangs
![image](https://user-images.githubusercontent.com/18963158/45481121-5a0bba80-b768-11e8-86d9-5f8c1425454f.png)

This is just one example, it happens almost on every such site.

**Comment by bsclifton** [comment]
@udion can you please try this with our [Developer channel release](https://brave.com/download-dev)? If this is still a problem there, I can help you create an issue

**Comment by udion** [comment]
@bsclifton 
I tried it on dev version. I think there is some problem with `google` option there as well. I also noticed that `facebook` option didn't open the popup with one click.

[SoundCloud site](https://soundcloud.com)
![image](https://user-images.githubusercontent.com/18963158/45589395-9561e100-b942-11e8-9408-b728a09f1903.png)

upon clicking sigin in
![image](https://user-images.githubusercontent.com/18963158/45589406-a7438400-b942-11e8-8ba4-cb1aff60f4f4.png)

when trying to click google option, it doesn't respond. However facebook option opens a popup for me to put in credentials.
This doesn't happen in other browser for instance mozilla.
